docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,122 @@
+    2. Check and install all the pre-requisites before executing edge-controller, which are
+        * docker (currently 18.06.0ce3-0~ubuntu) and check if service is up
+        * kubelet, kubeadm & kubectl (latest version)
+        * openssl (latest available in OS repos)
+    3. Generate certificates using openssl and save the certs in a predefined static path.
+    It will also compress the folder and display on the terminal so that user can pick it up and transfer it to edge node (VM/host) manually.
+    4. It will update the certificate information in `controller.yaml`
+    5. Start `kubeadm init`.
+
+       **NOTE:** If any issues or error reported from `kubeadm init`, `kubeedge cloud init` command shall be not responsible, as it may occur due to the environment. User has to resolve it. We are taking up this activity to have 1 click install approach for KubeEdge and not K8S.
+
+    6. Update `/etc/kubernetes/manifests/kube-apiserver.yaml` with below information
+    ```
+    - --insecure-port=8080
+    - --insecure-bind-address=0.0.0.0
+    ```
+
+    7. start edge-controller
+
+`kubeedge cloud reset`
+  - What is it? 
+    * This command will be responsible to bring down KubeEdge cloud components edge-controller and call `kubeadm reset` (to stop K8S)
+
+  - What shall be its scope ?
+    1. It shall get the nodes and execute `kubectl drain --force`.
+    2. Kill `edge-controller` process
+    3. Execute `kubeadm reset`
+
+**Edge commands:**
+
+`kubeedge node init`
+  - What is it? 
+    * This command will be responsible to install pre-requisites KubeEdge edge component (edge_core)
+
+  - What shall be its scope ?
+
+    1. Check version of OS and install subsequently the required pre-requisites using supported steps. Currently we will support **ONLY** (Ubuntu & CentOS)
+    2. Check and install all the pre-requisites before executing edge-controller, which are
+        * Docker (currently 18.06.0ce3-0~ubuntu) and check is service is up.
+        * mosquitto (latest available in OS repos) and check if running.
+        * kubelet
+    3. This command will take `--certPath` (string type) as mandatory option which shall be the certificates path; wherein the certs were transfered from cloud node and uncompressed. It will modify `$GOPATH/src/github.com/kubeedge/kubeedge/edge/conf/edge.yaml` file against `edgehub.websocket.certfile` and `edgehub.websocket.keyfile` fields.
+    4. Modify `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` and apply it using `kubectl` command to api-server
+
+`kubeedge node join`
+
+  - What is it? 
+    * This command will be responsible to make pre-requisite modifications and start KubeEdge edge component (edge_core)
+
+  - What it will do?
+
+    1. This command will take mandatory `-s` or `--server` flag to specify the address and port of the Kubernetes API server
```

Comment:
please give cli command with help as per previous comment.

Reply:
Have added the basic, required one's. Please check the updated document.